conf/zeppelin-site.xml.template

```diff
@@ -384,6 +384,12 @@
 </property>
 
 <property>
+  <name>zeppelin.owner.role</name>
```

Comment:
I feel the property name might not be very intuitive, how about calling it `zeppelin.notebook.default.owner.username` or something similar?

Reply:
Thank you for reviewing. It would be better, I think too. I'll fix it.